<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMorphology</class>
 <widget class="QWidget" name="DialogMorphology">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>404</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="tabletTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Morphology</string>
  </property>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>360</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>277</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Half width</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="slidera1">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>280</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal</string>
   </property>
   <property name="minimum">
    <number>-13</number>
   </property>
   <property name="maximum">
    <number>-1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="edita1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="OperationsGroup">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>40</y>
     <width>271</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Operations</string>
   </property>
   <widget class="QRadioButton" name="radioButtonErode">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>25</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Erode</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonDilate">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>25</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dilate</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonClose">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>55</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonOpen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>55</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonBlackhat">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>85</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Black hat</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonTophat">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>85</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Top hat</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonMedian">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>115</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Median Smooth</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonGradient">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>115</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gradient</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonExec">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>360</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Exec</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkBoxRealtime">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>367</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Realtime</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Source</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxSource">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>170</y>
     <width>271</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radioButtonCross">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cross</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonRect">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>32</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rect</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonEllipse">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>55</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ellipse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>140</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter size</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="slidera2">
    <property name="geometry">
     <rect>
      <x>103</x>
      <y>143</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal</string>
    </property>
    <property name="minimum">
     <number>-13</number>
    </property>
    <property name="maximum">
     <number>-1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="edita2">
    <property name="geometry">
     <rect>
      <x>204</x>
      <y>140</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>OperationsGroup</zorder>
  <zorder>pushButtonClose</zorder>
  <zorder>label_5</zorder>
  <zorder>slidera1</zorder>
  <zorder>edita1</zorder>
  <zorder>pushButtonExec</zorder>
  <zorder>chkBoxRealtime</zorder>
  <zorder>label_4</zorder>
  <zorder>comboBoxSource</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
